libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Recognizers/EntityRecognizers/Microsoft.ChannelMentionEntityRecognizer.schema

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": [ "implements(Microsoft.IRecognizer)" ],
```

Comment:
>implements [](start = 16, length = 10)

Do you want to implement IEntityRecognizer as well?  Or are you deprecating this?

Reply:
this isn't an entityrecognizer, it's only recgonizing data from the activity structure, not from the .text (or other entities itself).

---
In reply to: [511175574](https://github.com/microsoft/botbuilder-dotnet/pull/4848#discussion_r511175574) [](ancestors = 511175574)